src/set_theory/game/basic.lean

```diff
@@ -490,8 +491,15 @@ inductive inv_ty (l r : Type u) : bool → Type u
 | right₁ : l → inv_ty ff → inv_ty tt
 | right₂ : r → inv_ty tt → inv_ty tt
 
+instance (l r : Type u) [is_empty l] [is_empty r] : is_empty (inv_ty l r tt) :=
+⟨by rintro (_ | _ | _ | a | a); exact is_empty_elim a⟩
```

Comment:
Is this recommended mathlib style? I find it uglier than
```suggestion
⟨by rintro (_|_|_|a|a); exact is_empty_elim a⟩
```
(same below)

Reply:
I don't know what mathlib recommends here, but I've taken out the spaces.